.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: java
+sudo: false
+jdk:
+  - oraclejdk7
+install:
+  - wget http://www.eclipse.org/downloads/download.php?file=/eclipse/downloads/drops/R-3.6.2-201102101200/eclipse-SDK-3.6.2-linux-gtk-x86_64.tar.gz -O eclipse-3.6.2-linux64.tar.gz
+  - tar xzvf eclipse-3.6.2-linux64.tar.gz eclipse
+  - echo eclipsePlugin.dir=$(pwd)/eclipse/plugins > eclipsePlugin/local.properties
+script: ant build test
+notifications:
+  email:
+    recipients:
+      - skypencil+travis@gmail.com
```

Comment:
As I understand, every single build on Travis starts from the scratch. Thus probably we cannot use FindBugs binary built on Java 1.7 to test it with Java 1.8. Probably it would be good to enable build with Java 1.8 when some ant property is set (like `-DENABLE_JAVA_8_BUILD=true`). I guess it fails currently just because we should not accidentally publish FindBugs binaries which are built by Java-8. With some explicit property it should be ok.